internal/schemautil/custom_diff.go

```diff
@@ -153,3 +153,49 @@ func CustomizeDiffCheckStaticIpDisassociation(_ context.Context, d *schema.Resou
 	// TODO: Check that we block deletions that will result in too few static ips for the plan
 	return nil
 }
+
+// CustomizeDiffValidateFlinkJob is a custom diff validation function for flink jobs
+func CustomizeDiffValidateFlinkJob(_ context.Context, d *schema.ResourceDiff, m interface{}) error {
```

Comment:
I think we can move a `CustomizeDiffValidateFlinkJob` to the Flink package, since we are not using it anywhere else.